doc/source/user_guide/meshing_workflows.rst

```diff
@@ -607,4 +607,29 @@ Switch to solution mode
 
 .. code:: python
 
-    solver = meshing.switch_to_solver()
\ No newline at end of file
+    solver = meshing.switch_to_solver()
+
+Sample use of CommandArguments and accessor methods
+---------------------------------------------------
+This simple example shows how you use the CommandArgument attributes and explicit attribute accessor methods in a watertight geometry meshing workflow.
+Note: CommandArgument attributes are read-only.
+
+Import geometry
+~~~~~~~~~~~~~~~
+
+.. code:: python
+
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+
+import_filename = examples.download_file('mixing_elbow.pmdb', 'pyfluent/mixing_elbow')
+meshing = pyfluent.launch_fluent(mode="meshing", precision='double', processor_count=2)
+w = meshing.workflow
```

Comment:
Why are we suddenly getting these stupid comments?

Reply:
@seanpearsonuk , vale was having issues with spaces. That has been fixed now.